### DISCUSSION TITLE
Can't start a project

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#18893] [Help] [answered]
# Bug report

## Describe the bug
I'm trying to start a new project.
using npx create-next-app.
Everything downloads, and then I call npm run dev.
I immediately get about 100 errors:
*  There are multiple modules with names that only differ in casing.
   This can lead to unexpected behavior when compiling on a filesystem with other case-semantic.
*  Error: Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the 
   following reasons:
   1. You might have mismatching versions of React and the renderer (such as React DOM)
   2. You might be breaking the Rules of Hooks
   3. You might have more than one copy of React in the same app

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:
1.npx create-next-app client
2.cd client
3.npm run dev
4.Getting all the errors
5.If I open localhost:3000 I have an internal server error.

## Expected behavior
The app should start and I should see the default first page.

## Screenshots
![image](https://user-images.githubusercontent.com/49813173/98372394-5ae9bf80-2046-11eb-890d-ad9690bd550b.png)
![image](https://user-images.githubusercontent.com/49813173/98372407-65a45480-2046-11eb-9107-ef96b06c92d4.png)

## System information

- OS: [ Windows]
- Browser: chrome, firefox
- Version of Next.js: 10.0.1
- Version of Node.js: 13.12.0

**Comment by jamesmosier** [comment, marked as answer]
Hi @vekyoni. When changing directories via `cd` in Windows, make sure you are using the correct case. For example:

`cd C:\Users\vekyo\Desktop\MS\ticketing\client`

notice how I capitalized `Desktop` and it appears in your screenshot that you did not originally capitalize it.

**Reply by ghost** [reply]
Thanks!

**Reply by MacGyver98** [reply]
Thanks to both! @vekyoni for asking and @jamesmosier for giving us an answer.

Thas was turning me into a zombie!
